administrator/components/com_content/helpers/content.php

```diff
@@ -114,4 +114,26 @@ public static function countItems(&$items)
 
 		return $items;
 	}
+
+	/**
+	 * This function returns the id from the table __content based on title passed as parameter
+	 * 
+	 * @param   string  $item  title
+	 *
+	 * @return   integer
+	 */
+	public static function getarticleid($item)
```

Comment:
As I say you will going to remove it but just want to suggest you for better code style. You must follow code style here. It must be in camelCase. i.e this should be `getArticleId`